### PR TITLE
Move dev users to group_vars/all.yml

### PR DESCRIPTION
**Pull request by millerdev**
@snopoke cc @NoahCarnahan @esoergel

### CODE REVIEW

**Comment by dannyroberts** [comment]
nice!

**Comment by dannyroberts** [comment]
Good to merge after tests

**Comment by benrudolph** [comment]
love it

File: ansible/group_vars/all.yml [hunk 1]
```diff
@@ -19,7 +19,35 @@ shared_temp_dir_name: "temp"
...
+    - skelly
+    - sravfeyn
+    - wpride
+    - cchq
+  absent:
+    - preseed
+    - twymer
+    - tsheffels
+    - dmyung
+    - jenkins_slave
```
Comment (snopoke) [inline ansible/group_vars/all.yml]: We can remove these now that the playbook had been run

**Pull request merged**